sdk/storage/azure-storage-file-datalake/tests/test_file.py

```diff
@@ -907,6 +911,42 @@ def test_rename_file_with_non_used_name(self, datalake_storage_account_name, dat
         self.assertEqual(data, data_bytes)
         self.assertEqual(new_client.path_name, "newname")
 
+    @DataLakePreparer()
+    def test_file_encryption_scope(self, datalake_storage_account_name, datalake_storage_account_key):
```

Comment:
All my comments on `test_directory_encryption_scope` apply here too, including changing the name. :)

Reply:
Will make the necessary adjustments here as well to match the changes made in directory 😄